phpunit/bootstrap.php

```diff
@@ -45,6 +45,10 @@
 
 define('FIXTURE_DIR', __DIR__ . "/../tests/fixtures");
 
+// Check the resources state before trying to be sure that the tests are executed with up-to-date dependencies.
+require_once dirname(__DIR__) . '/src/Glpi/Application/ResourcesChecker.php';
```

Comment:
Not a problem, but why using `dirname` here? `__DIR__` alone is used in all include/require cases.

Reply:
Just to use the same as here: https://github.com/glpi-project/glpi/blob/6fb16060e2efbf4b879c43de895d1e1d3ab304da/public/index.php#L47-L48

`dirname(__DIR__)` is identical to `__DIR__ . '/..'`